application/Makefile

```diff
@@ -7,6 +7,9 @@ XARGS := xargs -0 $(shell test $$(uname) = Linux && echo -r)
 GREP_T_FLAG := $(shell test $$(uname) = Linux && echo -T)
 export PYFLAKES_BUILTINS=_
 
+OMIT := "*/tests/*,*/migrations/*,*apps.py,*asgi.py,*wsgi.py"
+OMIT :=$(OMIT)",*manage.py,*api/settings.py"
```

Comment:
Good idea to have that specified in a variable. Any reason not to have it in one line? Also, in the makefiles we don't need double quotes

Reply:
Will remove the double quotes
The length of omits line was getting long so I tried to split over two lines using '\' but it meant the omit on the line split was being missed. Is the way I've done this above the best around this issue ?